openrtb-core/src/main/protobuf/openrtb-ext.proto

```diff
@@ -220,6 +220,10 @@ message SKAdNetworkRequest {
   // Object containing the IABTL list definition.
   optional SKAdNetworkList skadnetlist = 4;
 
+  // Array of strings containing the supported SKAdNetwork versions. Always
+  // "2.0" or higher. Dependent on both the OS version and the SDK version.
+  repeated string versions = 5;
```

Comment:
Please add to the comment for `version` above, e.g.:
```
Note: With the release of SKAdNetwork 2.1, this field is deprecated in favor of the BidRequest.imp.ext.skadn.versions to support an array of version numbers.
```

Reply:
Added comments for `version` and also the `fidelities` related fields.